File: leetcode-submissions/539_Minimum_Time_Difference.cpp
class Solution {
public:
    int findMinDifference(vector<string>& timePoints) {
        vector<int>times;
        for(int i=0;i<timePoints.size();i++){
            int hrs=std::stoi(timePoints[i].substr(0,2));
            int minutes=std::stoi(timePoints[i].substr(3,5));
            times.push_back(minutes);
            
        }
    }
};

File: leetcode-submissions/539_Minimum_Time_Difference.ts
function findMinDifference(timePoints: string[]): number {
   const times:number[]=timePoints.map(time=>{
    const [hrs,minute]=time.split(":");
    return parseInt(hrs)*60+parseInt(minute);
   });
   times.sort();
   let minDiff:number=Number.MAX_SAFE_INTEGER;
   for(let i=0;i<times.length-1;i++){
minDiff=Math.min(minDiff,times[i+1]-times[i]);


   } 
minDiff=Math.min(minDiff,1440+times[0]-times[times.length-1]);
return minDiff;
};